examples/resources/wireguard_asymmetric_key/resource.tf

```diff
@@ -1,6 +1,9 @@
 resource "wireguard_asymmetric_key" "example" {
 }
 
+resource "wireguard_preshared_key" "example" {
```

Comment:
Let's put this in a new example - `examples/resources/wireguard_preshared_key/resource.tf` - which can then show it being used with `wireguard_asymmetric_key` too (I'd like to keep this one as the simplest the latter can be).

Reply:
done